src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -62,6 +64,41 @@ class _User:
     """The bidirectional channel to communicate with the user."""
 
 
+@dataclass
+class _CacheEntry:
+    """Represents an entry in the cache with expiry time."""
+
+    inv_bat_pair: InvBatPair
+    """The inverter and adjacent battery data pair."""
+
+    expiry_time: int
+    """The expiration time (taken from the monotonic clock) of the cache entry."""
+
+    @classmethod
+    def from_ttl(
+        cls, inv_bat_pair: InvBatPair, ttl: timedelta = timedelta(hours=2.5)
+    ) -> Self:  # type: ignore[name-defined] # pylint: disable=undefined-variable
```

Comment:
Why all these ignore?

Reply:
nope... didn´t know that exists :D. I'll update it right now

Reply:
updated